src/script.js

```diff
@@ -441,6 +461,7 @@ module.exports = {
   isMultisigOutput: isMultisigOutput,
   isNullDataOutput: isNullDataOutput,
 
+  parseMultisigScript: parseMultisigScript,
```

Comment:
I wonder if a `encode/decode/is(or check)` style module setup for each of these scripts types would be a great way to set it up?